my-app/src/pages/AboutUs.js

```diff
@@ -90,4 +98,4 @@ export default function AboutUs() {
       </div>
```

Comment:
I highly recommend making more components to hide away a lot of this code to make your `AboutUs.js` file much more cleaner and easier to read.

Think of it this way: someone goes to your `AboutUs.js` file because they want to know what's on the About Us page. So, in this file, you should provide a summary of what's on the page, but since there's a LOT of stuff on this page, you should avoid going into too much detail about any specific thing. You tell the reader "ok, on this page there's a theme switching button, a description, a carosel, a why us section, our social media, and a footer". Your return statement should look like this:
```
return (
   <div>
      <ThemeSwitcher />
      <Description />
      <Carousel />
      <WhyUs />
      <SocialMedia />
      <Footer />
   </div>
)
```

Then, if the reader is like "oh i want to know more about the carousel and how that works", they can look inside `Carousel.js` and get more details about the carousel component.

This makes it easier and faster for anyone looking at your code to understand what is going on. If someone is just trying to change one specific thing in one specific part of the about page, it's easier for them to find where to look when your `AboutUs.js` acts more as a table of contents, instead of literally all the contents and forcing the reader to scroll through the file themself to find the one thing they need to change.